packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -148,11 +149,16 @@ export class StripeHelper {
       useAdapter(this.redis);
     }
 
-    if (statsd) {
-      this.stripe.on('response', (response) => {
-        statsd.timing('stripe_request', response.elapsed);
+    this.statsd = statsd;
+    this.stripe.on('response', (response) => {
+      statsd.timing('stripe_request', response.elapsed);
+      // Note that we can't record the method/path as a tag
+      // because ids are in the path which results in too great
+      // of cardinality.
+      statsd.increment('stripe_call', {
+        error: (response.status >= 500).toString(),
```

Comment:
Not interested in 4xx errors?  Might be misleading for anyone just looking at the data in InfluxDB when the label is 'error'; they wouldn't know it's >= 500.

Reply:
Not really, mainly because it can happen easily and regularly, and just seeing a counter on a dashboard that it happened doesn't seem terribly useful. Ideally we want to know whether a large chunk of our requests are working correctly, or throwing errors. Debugging from a 4xx should be an error in Sentry.